ideascube/mediacenter/migrations/0013_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Document = app.get_model('mediacenter', 'Document')
+    db_alias = schema_editor.connection.alias
+    for document in Document.objects.using(db_alias).all():
+        document.summary = "<p>{}</p>".format(escape(document.summary))
```

Comment:
TinyMCE allow use to directly change the html dom inside a block element.
So, what we get at the end is the html text to directly use at display.
For now, we just have `p` and `a` elements (because we hide all other element and activate autolink plugin) But it can be any element `h1`, `img`, ...